public/stylesheets/site/2.0/02-elements.css

```diff
@@ -67,7 +67,6 @@ h2 {
   line-height: 1;
   margin: 0.429em 0;
   display: inline;
-    text-shadow: -1px -1px 2px #fff;
```

Comment:
Was this change meant to be in here? It seems unrelated to the rest and possibly needs different testing.